client/my-sites/sharing/controller.js

```diff
@@ -1,97 +1,77 @@
 /**
  * External Dependencies
  */
-var page = require( 'page' ),
-	React = require( 'react' ),
-	i18n = require( 'i18n-calypso' );
+import { createElement } from 'react';
+import page from 'page';
+import { translate } from 'i18n-calypso';
 
 /**
  * Internal Dependencies
  */
-var sites = require( 'lib/sites-list' )(),
-	utils = require( 'lib/site/utils' ),
-	notices = require( 'notices' ),
-	route = require( 'lib/route' ),
-	analytics = require( 'lib/analytics' ),
-	setTitle = require( 'state/document-head/actions' ).setDocumentHeadTitle,
-	analyticsPageTitle = 'Sharing';
-
+import notices from 'notices';
+import { pageView } from 'lib/analytics';
 import { renderWithReduxStore } from 'lib/react-helpers';
+import { sectionify } from 'lib/route';
+import Sharing from 'my-sites/sharing/main';
```

Comment:
Minor: Seems like in this and following imports it would be reasonable to import relative:

```js
import Sharing from './main';
```